config/initializers/sentry.rb

```diff
@@ -1,5 +1,6 @@
 if ENV.fetch('SENTRY_DSN', nil).present?
   Sentry.init do |config|
+    config.environment = HostEnv.env_name
```

Comment:
This might be erroring out due to HostEnv not being loaded at the point Sentry is inited. You could try `ENV.fetch('ENV')`